Toolchain/BuildGDB.sh

```diff
@@ -82,7 +82,7 @@ pushd "$DIR/Tarballs"
         md5="$($MD5SUM "$GDB_PKG" | cut -f1 -d' ')"
     fi
     if [ "$md5" != "$GDB_MD5SUM" ]; then
-        curl -C - -O "$GDB_BASE_URL/$GDB_PKG"
+        curl -C - -LO "$GDB_BASE_URL/$GDB_PKG"
```

Comment:
This looks like it should be in the previous commit?

Reply:
I added the `-L` flag because `curl` doesn't follow HTTP 3xx redirects by default (which is what ftpmirror.gnu.org uses to redirect to the right mirror)